ports/winit/Cargo.toml

```diff
@@ -27,7 +27,7 @@ ProductName = "Servo"
 
 [features]
 debugmozjs = ["libservo/debugmozjs"]
-default = ["webdriver", "max_log_level"]
+default = ["webdriver", "max_log_level", "media-gstreamer", "native-bluetooth", "egl", "layout-2020"]
```

Comment:
Is there any way to make "egl" dependent on whether or not the build is a "uwp" build? If so we could remove this code:

https://github.com/servo/servo/blob/7aaad0aa7eac2d0c98ff17091013fda77359397d/python/servo/command_base.py#LL857C1-L862C39

Reply:
Perhaps using something like this:

https://github.com/rust-lang/cargo/issues/1197#issuecomment-268203727

Reply:
Do we even have uwp builds, as far as I know we removed them from CI, because there was no interest in them.